cmd/podman/version.go

```diff
@@ -43,6 +44,9 @@ func versionCmd(c *cliconfig.VersionValues) error {
 
 	versionOutputFormat := c.Format
 	if versionOutputFormat != "" {
+		if strings.Replace(versionOutputFormat, " ", "", -1) == "{{json.}}" {
```

Comment:
Sadly this will not remove tabs.
How about
```
 strings.Join(strings.Fields(versionOutputFormat),"")
```

Reply:
yup, indeed